src/libos/src/net/socket/unix/stream/address_space.rs

```diff
@@ -7,9 +7,28 @@ lazy_static! {
     pub(super) static ref ADDRESS_SPACE: AddressSpace = AddressSpace::new();
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub enum AddressSpaceKey {
+    FileKey(usize),
+    AbstrKey(String),
+    NoExist,
+}
+
+impl AddressSpaceKey {
+    pub fn new(file_key: Option<usize>, abstr_key: Option<String>) -> Self {
```

Comment:
Separate the `new` into `AddressSpaceKey::from_inode` and `AddressSpaceKey::from_path` to make it more clear for the initialization of it.

Reply:
Done.